src/errors.rs

```diff
@@ -15,6 +15,18 @@ pub enum MpesaError {
     C2bSimulateError(serde_json::Value),
     #[error("Account Balance request failed: {0}")]
     AccountBalanceError(serde_json::Value),
+    #[error("Bill manager onboarding failed: {0}")]
+    OnboardError(serde_json::Value),
+    #[error("Bill manager onboarding modify failed: {0}")]
+    OnboardModifyError(serde_json::Value),
+    #[error("Bill manager bulk invoice failed: {0}")]
+    BulkInvoiceError(serde_json::Value),
+    #[error("Bill manager reconciliation failed: {0}")]
+    ReconciliationError(serde_json::Value),
+    #[error("Bill manager single invoice failed: {0}")]
+    SingleInvoiceError(serde_json::Value),
+    #[error("Bill manager cancel invoice failed: {0}")]
+    CancelInvoiceError(serde_json::Value),
```

Comment:
If #67 is merged before this PR, this file to be changed to reflect the changes introduced in that PR

Reply:
#67 is merged already, you can go ahead and rebase to apply the new changes

Reply:
Tests failing because of that change yeah?